### PR TITLE
chore: documentation for InputController

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Added documentation to InputController

### CODE REVIEW

**Review by sandrade-dcl — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs [hunk 1]
```diff
@@ -4,6 +4,9 @@
 using UnityEngine;
 using UnityEngine.EventSystems;
 
+/// <summary>
+/// Mapping for Trigger actions
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs:8]: Here we are missing an opportunity for explaining how trigger actions work. The same for other action types.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs [hunk 2]
```diff
@@ -226,6 +250,11 @@ public enum Modifier
         FocusNotInInput = 0b0000010,
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs]: Remember we had to answer how the modifiers work, each one of these deserves an explanation.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs [hunk 3]
```diff
@@ -256,6 +290,13 @@ public static bool PassModifiers(Modifier modifiers)
         return true;
     }
 
+    /// <summary>
+    /// Process an input action mapped to a keyboard key.
+    /// </summary>
+    /// <param name="action">Trigger Action to perform</param>
+    /// <param name="key">KeyCode mapped to this action</param>
+    /// <param name="modifierKeys">KeyCodes required to perform the action</param>
+    /// <param name="modifiers">Miscellaneous modifiers required for this action</param>
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs:300]: What are miscellaneous modifiers? Can we provide an example?

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs (hunk 3 above)
Comment (AjimenezDCL) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs:300]: They can be anything: having renderer enabled, mouse locked, not jumping or whatever you can imagine. I will provide a more clear description

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs:8]: I actually had the documentation for the actions there and then I decided (and later forgot about it) that it was better to put it in their classes. 

Gonna do it!

**Review by BrianAmadori — APPROVED** [review]

**Pull request merged**